src/InputToolbox.js

```diff
@@ -36,11 +36,11 @@ export default class InputToolbox extends React.Component {
                     </div>
                 </div>
                 <div class="row justify-content-center">
-                    {Object.values(this.dialogs).map(dialog => (
-                        <div class={"col-lg-4" + (this.state.openDialog === dialog ? "" : " d-none d-lg-block")}>
```

Comment:
Notice the class is assigned as `d-none d-lg-block` when it is not the "open" dialog.